### DISCUSSION TITLE
Put # of machines on map pins

### DISCUSSION THREAD

**Discussion by agscala** [pinballmap/pbm#1539] [Ideas]
It'd be nice if I didn't need to mindlessly open every location just to find the ones that have a bunch of machines available.

AFAIK Google maps let's you customize the look of the pins. It would be better if it showed the number of machines at that location

**Comment by RyanTG** [comment]
We do? 
![image](https://github.com/pinballmap/pbm/assets/632682/a122cad0-3c28-4edc-bf29-a251949578a8)

**Comment by RyanTG** [comment]
If you are referring to the Android app (you are commenting on the website repo right now), then please see https://github.com/pinballmap/pbm-react/issues/476

In the meantime, clicking "List" might help a little.

**Comment by RyanTG** [comment]
This is addressed now (for a while now).
